projects/1.16/assets/the-afterlight/the_afterlight/patchouli_books/afterlight_tome/zh_cn/entries/chapters/category_1_entry_5.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "传送门",
+    "icon": "the_afterlight:moonstone_2",
+    "category": "the_afterlight:category_1",
+    "sortnum": 5,
+    "pages": [
+        {
+            "type": "patchouli:text",
+            "text": "和先前提到的一样，通往隐光之域的传送门是由怪异黑曜石搭建且需月光之杖开启的。首先，搭建一个类似下界传送门造型的怪异黑曜石框；然后用月光之杖点亮它即可。神秘的隐光之域就在那扇传送门后，等着你前去探索。"
+        },
+        {
+            "type": "patchouli:relations",
+            "entries": ["the_afterlight:category_2_entry_1", "the_afterlight:category_3_entry_1"],
+            "text": "这部秘典会在将来有需要时继续提供帮助。后面的章节记录了更多有关隐光之域的知识，将在进入隐光之域后开放阅读。",

```

```suggestion
            "text": "这部秘典会在将来有需要时继续提供帮助。后面的章节记录了更多有关隐光之域的知识，将在进入隐光之域后解锁。",

```
这里Patchouli用的是解锁，所以我认为解锁好点